meeting-notes/2018/2018-06-25--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,157 @@
+ - Blocked: No
+ - Next:
+   - More reviews of IPLD Format changes implementation from @richardschneider
+   - Writing down what the essential core of GraphSync is from my perspective
+   - Playing around with js-libp2p with GraphSync in mind
+
+@hugomrdias
+ - Done:
+   - Implemented non-leveldb indexed db datastore
+   - secio/crypto testing
+ - Blocked:
+ - Next:
+   - Q2 retro and Q3 planning
+   - make web crypto work with local daemon
+   - Test/fix add file with more than 256mb with the ipfs api [#788](https://github.com/ipfs/js-ipfs-api/issues/788#issuecomment-398505668)
+ 
+@fsdiogo
+- Done:
+    - Better WebSocket errors
+        - https://github.com/libp2p/js-libp2p-websocket-star/pull/54
+    - Issue with a “roadmap” to track and keep progress of issues about errors and daemon crashes
+        - https://github.com/ipfs/js-ipfs/issues/1406
+- Blocked:
+    - N/A
+- Next:
+    - Create a repo that gathers the error codes in the `js-ipfs` scope
+        - https://github.com/ipfs/js-ipfs/issues/1406#issuecomment-399495062
+    - Create PRs in repos that are using error codes to import the new repo
+    - Analyze the `WebUI` and `IPFS Companion` current state to propose OKRs for the next quarter (In Web Browsers WG)
+
+@vasco-santos
+  - Done:
+    - PR for `js-ipfs` with initial working implementation of `IPNS working locally`
+      - https://github.com/ipfs/js-ipfs/pull/1400
+    - Created `js-ipns` module (same as go-ipns), but name is under discussion
+      - https://github.com/ipfs/js-ipns/pull/1
+    - Merged 2 PRs on `js-ipfsd-ctl`
+      - https://github.com/ipfs/js-ipfsd-ctl/pull/270
+      - https://github.com/ipfs/js-ipfsd-ctl/pull/268
+  - Blocked:
+    - Define IPNS modules and naming with go
+      - https://github.com/ipfs/js-ipfs/pull/1400
+  - Next:
+    - Add nanoseconds precision on IPNS records validity for interoperability with go
+      - https://github.com/ipfs/js-ipns/pull/1
+    - Finish test for IPNS
+      - https://github.com/ipfs/js-ipfs/pull/1400
+    - Have IPNS modules defined and "js-ipns" module released
+    - Create WIP PR for IPNS Specs
+    - `libp2p` Q3 OKRs
+
+@jacobheun
+  - Done:
+    - Added interop tests for private network. Fixed an issue with dialing order for go.
+    - Found an issue with bad private network connections (such as dialing to a public network) where the connection would hang until timeout, 60s. This is due to https://github.com/dominictarr/pull-reader/issues/5, I am working on a fix for pull-reader, which will include updates to pull-length-prefixed.
+    - Demo'd private network interop at the [IPFS All Hands today](https://youtu.be/fObld4alGag?t=47).
+  - Blocked: 
+    - N/A
+  - Next:
+    - Fix issue with pull-reader handling overreading
+    - Finish submitting PRs for private networking
+    - Work on Q2 retro and Q3 OKRs for libp2p
+    - Update examples for libp2p config update and help get that merged and released
+
+@achingbrain
```

Comment:
Any demo/tutorial on using mfs with js-ipfs? It would be great to add one to the collection! :)

Reply:
Thinking more about this. A lot of people don't know or are confused about MFS, having a tutorial will go a long way. Can you make this happen, @achingbrain ?